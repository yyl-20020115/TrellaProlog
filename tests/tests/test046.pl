:- initialization(main).
:- use_module(library(lists)).

main :-
	append([a,b],[c,d],L), write(L), nl,
	append(L1,L2,[a,b,c,d]), write(L1), write(' <==> '), write(L2), nl, fail.
main.
