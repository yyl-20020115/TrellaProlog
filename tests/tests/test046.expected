"abcd"
[] <==> "abcd"
"a" <==> "bcd"
"ab" <==> "cd"
"abc" <==> "d"
"abcd" <==> []
